### PR TITLE
LPS-200562 | Add a testcase to support export object entries at site level

### PR DESCRIPTION
**Pull request by Hijky**
Background:
Add a testcase to support export object entries at site level

Test Plan
CanSelectAllExistingFields
When selecting 'C_Student (v1_0 - Liferay Object REST)' as Entity Type in Data Migration Center --> Export File
Then the list of fields to select contains all the fields available with GET request

CanDownloadFileWithAllFieldsSelected
And Given 'C_Student (v1_0 - Liferay Object REST)' as Entity Type  and current site as Scope in Data Migration Center --> Export File
When exporting file in JSON format with all fields selected
Then all fields values available in the downloaded file

CanSeeCreatedSiteInScopeSelector
And Given a new site named test
And Given 'C_Student (v1_0 - Liferay Object REST)' as Entity Type and default site as scope in Data Migration Center --> Export File
When click the scope selector
Then test site present in the dropdown list

CanSelectGlobalSiteAsScope
When 'C_Student (v1_0 - Liferay Object REST)' as Entity Type in Data Migration Center --> Export File
Then and Global site available as scope

CanExportSpecificFieldsWithJSONTConfigurationNode
And Given 'C_Student (v1_0 - Liferay Object REST)' as Entity Type and current site as Scope in Data Migration Center --> Export File
When exporting file in JSONT format with field 'name' selected
Then 'name' value for created entry, actions node, configuration node available in the dowloaded file

CannotSeeScopeSelectorForObjectDefinitionsScopedByCompany
Given company scoped University object definition with text field 'name' created
When 'C_University (v1_0 - Liferay Object REST)' as Entity Type in Data Migration Center --> Export File
Then 'Scope' selector is not present

Jira ticket:
https://liferay.atlassian.net/browse/LPS-200562

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtSiteLevel.testcase [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+		property portal.acceptance = "true";
+
+		task ("When selecting 'C_Student (v1_0 - Liferay Object REST)' as Entity Type in Data Migration Center --> Export File") {
+			ImportExport.selectEntity(entityType = "C_Student (v1_0 - Liferay Object REST)");
+		}
+
+		task ("Then the list of fields to select contains all the fields available with GET request") {
+			var fieldList = "name,actions,creator,dateCreated,dateModified,externalReferenceCode,id,keywords,scopeKey,status";
+
+			var exportFields = ${fieldList};
```
Comment (yanan) [inline modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtSiteLevel.testcase:236]: please update to remove this, thanks

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtSiteLevel.testcase [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+				en_US_label = "StudentAccounts",
+				name = "studentAccounts",
+				objectDefinitionId1 = ${studentId},
+				objectDefinitionId2 = ${accountId},
+				type = "oneToMany");
+		}
+
+		task ("And Given object entries of Student, Subject, Account created with related entries") {
+			var accountEntryId = AccountAPI.createAccount(name = "accountForStudent");
+			var studentEntryId = ObjectDefinitionAPI.createObjectEntryWithName(
```
Comment (yanan) [inline modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtSiteLevel.testcase:59]: please help check if we can create object entry and its related entry at the same time to reduce some code, thanks

Comment (Hijky) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtSiteLevel.testcase:59]: For this question, I have tested locally and it has some errors, I have asked it in the channel and I think we can keep it

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Review by yanan — APPROVED** [review]

**Comment by Hijky** [comment]
ci:forward

**Pull request closed without merging**